gradio/components/chatbot.py

```diff
@@ -55,6 +55,7 @@ def __init__(
         show_share_button: bool | None = None,
         show_copy_button: bool = False,
         avatar_images: tuple[str | Path | None, str | Path | None] | None = None,
+        disable_html_sanitization: bool = False,
```

Comment:
Would a simpler name for this parameter be:
```suggestion
        sanitize_html: bool = True,
```